tests/test_wave_01.py

```diff
@@ -47,16 +51,22 @@ def test_get_task(client, one_task):
     }
 
 
+@pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_task_not_found(client):
     # Act
     response = client.get("/tasks/1")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
-    assert response_body == None
 
+    raise Exception("Complete test with assertion about response body")
```

Comment:
Update wave 1 & 3 project docs to explicitly include that they need to complete these tests as part of finishing the wave

Reply:
Nevermind, I totally forgot you made a separate PR that addresses these